cpp/include/cudf/strings/detail/json.hpp

```diff
@@ -32,6 +32,7 @@ namespace detail {
 std::unique_ptr<cudf::column> get_json_object(
   cudf::strings_column_view const& col,
   cudf::string_scalar const& json_path,
+  get_json_object_options options     = get_json_object_options{},
   rmm::cuda_stream_view stream        = rmm::cuda_stream_default,
```

Comment:
Typically I don't set default parameters for the `cudf::detail::` API, because that can lead to confusion. You can call the `cudf::` and `cudf::detail::` versions by the same list of arguments. For example: 
```
cudf::strings::get_json_object(col, json_path);
cudf::strings::detail::get_json_object(col, json_path);
```

So, if possible, please remove default parameters. By doing so, if you want to call `get_json_object(col, json_path)` then you can call the `cudf::strings::` version, and if you want to call `get_json_object(col, json_path, options,...)` then you must call the `cudf::strings::detail::` version.

Reply:
Done